pkg/test/resourcefixture/testdata/basic/run/v1beta1/runjob/jobwithsecretmanagersecret/_generated_object_jobwithsecretmanagersecret.golden.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: run.cnrm.cloud.google.com/v1beta1
```

Comment:
Nit: it would be nice to capture the output before we turned on the direct controller!  Maybe we should encourage test changes to be in a separate PR?